pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -290,6 +290,18 @@ type MachineConfig struct {
 	//    - value: machineSeccompExample()
 	MachineSeccompProfiles []*MachineSeccompProfile `yaml:"seccompProfiles,omitempty" talos:"omitonlyifnil"`
 	//  description: |
+	//    Override (patch) settings in the default OCI runtime spec for CRI containers.
+	//
+	//    It can be used to set some default container settings which are not configurable in Kubernetes,
+	//    for example default ulimits.
+	//    Note: this change applies to all newly created containers, and it requires a reboot to take effect.
+	//  examples:
+	//    - name: override default open file limit
+	//      value: machineBaseRuntimeSpecOverridesExample()
+	//  schema:
+	//    type: object
+	MachineBaseRuntimeSpecOverrides Unstructured `yaml:"baseRuntimeSpecOverrides,omitempty"`
```

Comment:
does this apply for both cri and containerd? I guess we only need for cri?

Reply:
only for CRI